projects/packages/connection/src/class-tokens.php

```diff
@@ -499,9 +499,10 @@ public function disconnect_user( $user_id, $can_overwrite_primary_user = false )
 	 * Able to select by specific capability.
 	 *
 	 * @param string $capability The capability of the user.
+	 * @param bool   $is_any_user_connected Set to true to return when a single connected user is found.
 	 * @return array Array of WP_User objects if found.
 	 */
-	public function get_connected_users( $capability = 'any' ) {
+	public function get_connected_users( $capability = 'any', $is_any_user_connected = false ) {
```

Comment:
Thanks again for the PR.

I have one suggestion. What do you think to name this argument a bit more generic and make it fit better with the method purpose. When we say `is_any_user_connected` we are speaking about a very specific use of this method, which may not make sense for someone looking at it.

Let's make it a `$limit` parameter, where you can set how many users you want to get at most. By default, it gets all users, but in `has_connected_user` we can pass `1` and be satisfied.

Also, let's add some unit tests. If you make the change I can help by writing the tests

Reply:
Hi Leo, good point a limit definitely makes more sense for the method.

Just made those changes.